docs.go

```diff
@@ -0,0 +1,289 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/cpuguy83/go-md2man/md2man"
```

Comment:
❓ _(blocking question)_

I think that since you added modules support, this goes into the `go.mod` file now?

Reply:
Yes, rebased on top of the latest master and added it to `go.mod`.